localdev/scripts/binoculars-start.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# this will update kind kube config copied from host to work inside docker
+if [ -f /.dockerenv ]; then
+    DOCKER_HOST=$(ping -c 1 host.docker.internal | grep -m 1 -o -E '[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}')
```

Comment:
host.docker.internal has native support to only windows and mac by default, and for linux the Docker version installed needs to be relatively new. We probably want to document all this? https://medium.com/@TimvanBaarsen/how-to-connect-to-the-docker-host-from-inside-a-docker-container-112b4c71bc66

Reply:
Also, I do not 100% understand why this is needed. Is this to make sure kubectl points to the correct place?

Reply:
yes, this is so armada services that talk to kubernetes control plane can do so using .kube/config of the host computer -- and they need to use a host name that matches the tls cert. Will add to comment (and extract for reuse)